todo_app/app.py

```diff
@@ -1,14 +1,41 @@
-from flask import Flask
+
+from flask import Flask,request, render_template,redirect,url_for
+
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+from todo_app.data.session_items import get_item
+from todo_app.data.session_items import save_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+   
+	return render_template("index.html",items = get_items())
+	
+@app.route('/create', methods =['POST','GET'])
```

Comment:
You app doesn't actually make GET requests to /create so it shouldn't be in the list of supported methods